build-tools/automation/azure-pipelines-oss.yaml

```diff
@@ -103,8 +103,8 @@ stages:
       displayName: make jenkins
 
     - script: >
-        echo "make create-installers V=1 CONFIGURATION=$(XA.Build.Configuration)" &&
```

Comment:
Don't we want to *continue* calling `make create-installers`, so that the new `make create-workload-installers` target is executed?

Reply:
…or do we not want to create a Workload installer for the OSS pipeline?

Reply:
We would have to add commands to create the `.nupkg` files for `create-installers` to work on the OSS pipeline. I would also have to make a Windows phase to create the `.msi`.

I thought it was simpler to not create these installers on the OSS pipeline for now. But if we have a need, we could?

Reply:
>  I would also have to make a Windows stage to create the `.msi`.

We don't have a Windows stage OSS build (but perhaps we should?), so I don't think this would be needed.

I'm largely "just pondering" the community engagement angle.  Is there a reason to *not* provide "OSS-blessed" packages, and/or *ensure* that the OSS tree *can* produce the `.nupkg` files?

That said, it might very well fail *now*, as the `.nupkg` files likely pulls in proprietary files, so this should be done as a separate PR, if ever.